tests/_utils/view.js

```diff
@@ -0,0 +1,836 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+/* jshint latedef:false */
```

Comment:
E.g. I already caught myself three times on reading https://github.com/ckeditor/ckeditor5-engine/blob/master/src/treemodel/schema.js#L23 as `Schema` because I was expecting `Schema` to be defined first.